moped-editor/src/components/GridTable/Filters.js

```diff
@@ -676,38 +745,34 @@ const Filters = ({
           </Button>
         </Grid>
         <Grid item xs={12} md={1}>
-          {Object.keys(filterParameters).length > 0 && (
-            <Button
-              className={classes.bottomButton}
-              fullWidth
-              variant="outlined"
-              startIcon={<BackspaceOutlinedIcon />}
-              onClick={handleClearFilters}
-            >
-              Reset
-            </Button>
-          )}
+          <Button
+            className={classes.bottomButton}
+            fullWidth
+            variant="outlined"
+            startIcon={<BackspaceOutlinedIcon />}
+            onClick={handleClearFilters}
+          >
+            Reset
+          </Button>
         </Grid>
         <Hidden mdDown>
           <Grid item xs={12} md={7}>
             {""}
           </Grid>
         </Hidden>
         <Grid item xs={12} md={2}>
-          {Object.keys(filterParameters).length > 0 && (
```

Comment:
Same here, the `Grow` component below has a prop that uses `handleApplyValidation` to control whether it grows in (visible) or not (hidden). The `handleApplyValidation` helper also creates feedback strings about why filters are not valid that we don't display anywhere.

Reply:
nice!